pkg/lib/DynamicListForm.jsx

```diff
@@ -19,10 +19,15 @@ export class DynamicListForm extends React.Component {
         this.onItemChange = this.onItemChange.bind(this);
     }
 
-    removeItem(idx, field, value) {
+    removeItem(idx) {
         this.setState(state => {
+            const validationFailedDelta = this.props.validationFailed ? [...this.props.validationFailed] : [];
+            validationFailedDelta.splice(idx, 1);
+            this.props.onValidationChange(validationFailedDelta);
```

Comment:
Please call with `?.` to get along with an absent onValidationChange property. This is an exported class, so I think it's better to make that new property optional.